app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_post, only: [:new, :create]
+  
+  def new
+    redirect_to post_path(@post.id)
+  end
+  
+  def create
+    @comment = @post.comments.new(comment_params)
+    if @comment.save
+      redirect_to post_path(@post.id)
+    else
+      flash[:alert] = 'コメントを(140文字以内で)入力してください。'
+      redirect_to post_path(@post.id)
```

Comment:
ここはリダイレクトよりrenderで記述してあげた方が良いです！
`redirect_to`だと、記述してた内容がクリアされてしまうのに対して、renderだと内容を保持したままページに戻れます。

```suggestion
      render 'posts/show'
```